employees/common/constants.py

```diff
@@ -0,0 +1,13 @@
+from django.utils.translation import ugettext_lazy
+from decimal import Decimal
+
+
+class ReportModelConstants:
```

Comment:
I'm wondering why is this a class? Maybe enum would be more appropriate? What do you think?

Reply:
So the only change in here would be defining Enum inheritance? 😛

Reply:
Okay, obviously more than that, because it stopped working 😱

Reply:
https://docs.python.org/3/library/enum.html#creating-an-enum
But you can leave it like it is if you have any argument that this should be a class. Like: I want to inherit from it. Or: I want to create instances.

Reply:
Done